cli/command/service/create.go

```diff
@@ -74,6 +74,8 @@ func runCreate(dockerCli *command.DockerCli, opts *serviceOptions) error {
 
 	ctx := context.Background()
 
+	resolveServiceImageDigest(dockerCli, &service)
```

Comment:
Changed to fail the "service create" or "service update" command if digest resolution fails when using content trust.